data cleaning.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Nice, thanks for merging reference and citizen data cleaning!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/pitne-season-watch/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>